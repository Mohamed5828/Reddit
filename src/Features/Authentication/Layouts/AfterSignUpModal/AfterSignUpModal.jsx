import Modal from "react-bootstrap/Modal";

import { MyModal } from "./AfterSignUpModal.styled";

import AfterSignUp from "../AfterSignUp/AfterSignUp";

/**
 * AfterSignUpModal component that is used to show the modal of AfterSignUp
 * @returns {React.Component} AfterSignUpModal component that is used to show the modal of AfterSignUp
 */

const AfterSignUpModal = ({ ...props }) => {
  return (
    <MyModal
      {...props}
      size="lg"
      aria-labelledby="contained-modal-title-vcenter"
      centered
      backdrop={"static"}
    >
      <Modal.Body>
        <AfterSignUp setModalAfterSignUp={props.setModalAfterSignUp} />
      </Modal.Body>
    </MyModal>
  );
};

export default AfterSignUpModal;
