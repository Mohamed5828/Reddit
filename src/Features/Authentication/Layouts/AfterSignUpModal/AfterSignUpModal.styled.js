import styled from "styled-components";
import Modal from "react-bootstrap/Modal";

const fontFamily = "'IBM Plex Sans', sans-serif;";


export const MyModal = styled(Modal)`
.modal-content {
  box-shadow: 0px 0px 10px 3px #8080808a;
  height: 600px;
  width: 90%;
  margin: auto;
  max-width: 380px;
}

> div {
  min-width:350px;
}
`;

export const MyModalHeader = styled(Modal.Header)`
  border: none;

  button {
    font-size: 10px;
  }
`;
export const HaveAccount = styled.span`
padding: 0 0 0 33px;
  font-size: 12px;
  margin-bottom: 100px;

  button {
    padding: 0;
    font-weight: 700;
    border: none;
    background-color: transparent;
    text-decoration: underline;
    color: ${({ theme }) => theme.color.link};
    font-family: ${fontFamily};
  }
`;
