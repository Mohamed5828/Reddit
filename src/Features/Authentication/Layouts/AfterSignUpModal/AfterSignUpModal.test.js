import { shallow } from "enzyme";

import AfterSignUpModal from "./AfterSignUpModal";

describe("AfterSignUpModal Component", () => {
  it("this is a test for AfterSignUpModal Component", () => {
    expect(shallow(<AfterSignUpModal />)).toMatchSnapshot();
  });
});
