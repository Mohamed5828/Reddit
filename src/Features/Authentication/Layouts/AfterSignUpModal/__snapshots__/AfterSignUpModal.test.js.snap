// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AfterSignUpModal Component this is a test for AfterSignUpModal Component 1`] = `
<Styled(Modal)
  aria-labelledby="contained-modal-title-vcenter"
  centered={true}
  size="lg"
>
  <ModalBody>
    <AfterSignUp />
  </ModalBody>
</Styled(Modal)>
`;
